src/configuration.ts

```diff
@@ -120,10 +120,10 @@ function findNextVersion(rootPath: string): string | undefined {
 export function findRepoFromPkg(pkg: any): string | undefined {
   const url = pkg.repository.url || pkg.repository;
   const normalized = normalize(url).url;
-  const match = normalized.match(/github\.com[:/]([^./]+\/[^./]+)(?:\.git)?/);
+  // reference https://www.debuggex.com/r/H4kRw1G0YPyBFjfm
+  const match = normalized.match(/([\/\/]?github\.com[\/:])([\w\.@\:\/\-~]+[^.git])/);
```

Comment:
what do you think about `github\.com[:/]([^/]+\/[^/]+?)(?:\.git)?$`? it's a smaller change and should work fine for the above test case too :)

Reply:
I can try that, @Turbo87

Reply:
@Turbo87 Updated the regex. Can you check and approve if it looks good.